### DISCUSSION TITLE
Python Plugins in Mason don't work

### DISCUSSION THREAD

**Discussion by kvlx-alt** [mason-org/mason.nvim#1372] [Q&A]
I'm using Neovim with Mason on Fedora. I wanted to try out some Python plugins, but none of them are working for me. I installed python-lsp-server, and it seems to install correctly, but it doesn't show anything in a .py file. It's as if it doesn't recognize it. I tried installing lua-language-server, and it works perfectly. I also tried other plugins, and they work fine, but for some reason, these Python plugins get installed but do anything.
![2023-06-28-020108_1299x463_scrot](https://github.com/williamboman/mason.nvim/assets/118694485/29985b69-5841-4484-a73d-470e538c2132)
This work with lua-language-server
![2023-06-28-015925_901x489_scrot](https://github.com/williamboman/mason.nvim/assets/118694485/858f26e1-b9d7-4b29-88f7-5a939ecc22bb)

**Comment by yutkat** [comment]
Please paste `LspInfo` as well.
